kitchen-tests/kitchen.yml

```diff
@@ -12,6 +12,7 @@ provisioner:
   name: dokken
   client_rb:
     diff_disabled: true
+  chef_license: "accept-no-persist"
```

Comment:
Is this just in case someone runs this locally? Otherwise the env var should cover it.